sublime-package.json

```diff
@@ -26,6 +26,17 @@
               "type": "string",
               "deprecationMessage": "Use \"languages\" instead."
             },
+            "ClientCommand": {
+              "type": "array",
+              "default": [
+                "path/to/language-server-binary",
+                "--some-option"
+              ],
```

Comment:
I recently learned that vscode-json-language-server can also inserts snippets. Can we add some sweet snippet for the `"command"` completion? https://code.visualstudio.com/Docs/languages/json#_define-snippets-in-json-schemas

Reply:
but `default` acts as a snippet also. It's automatically inserted when adding that key.

Reply:
Yes but the `"path/to/language-server-binary`" and `"--some-option"` are not snippet fields. But maybe suitable for another pr.